src/Application/Common/Interfaces/IActivityFeedService.cs

```diff
@@ -0,0 +1,8 @@
+using SSW.Rewards.Shared.DTOs.ActivityFeed;
+
+namespace SSW.Rewards.Application.Common.Interfaces;
+
+public interface IActivityFeedService
+{
+    public Task<IList<ActivityFeedViewModel>> GetActivities(ActivityFeedFilter filter, int skip, int take, CancellationToken cancellationToken);
```

Comment:
I don't see the value of this abstraction. I think the EF query could just be written in the handler.

Reply:
Agree.